scala/providers.bzl

```diff
@@ -26,9 +26,8 @@ def create_scala_provider(ijar, class_jar, compile_jars,
   )
 
 ScalacProvider = provider(
-    doc = "ScalaProvider",
+    doc = "ScalaCProvider",
```

Comment:
We use ScalacProviser for the provider but capitalize the C here.

Reply:
Indeed. In general I mix scalac provider and scalac repositories provider.  Do you think that is ok?

Reply:
done